data/notifications/x-nemo.call.missed.preview.conf

```diff
@@ -0,0 +1,6 @@
+appIcon=icon-lock-missed-call
+transient=true
+x-nemo-preview-icon=icon-lock-missed-call
+x-nemo-feedback=call
+x-nemo-priority=120
+x-nemo-led-disabled-without-body-and-summary=false
```

Comment:
Yes.  In fact, the preview may soon be removed from missed-call notifications entirely, since it's unlikely that the preview tells you anything if you somehow missed the incoming call dialog.